src/node/node_class.hpp

```diff
@@ -195,7 +195,11 @@ class ObjectWrap {
 	static bool has_native_method(const std::string& name);
 
   private:
-    static ClassType s_class;
+    //These wrapper methods exists to force the compiler to initialize these class template static members in sequence before they are used
```

Comment:
This comment isn't correctly aligned, and it has the same issue I describe below. I'm also not sure it is helpful.

Reply:
will remove it